src/data/uma-nickname-requirements.csv

```diff
@@ -140,3 +140,4 @@
 "全レースで2着以内で、\n菊花賞、天皇賞（春）、宝塚記念を勝利し、\n目標を全て達成し育成を完了する","<size=22>Finish ≥2nd in every single race\nWin the Kikka Sho, Tenno Sho (Spring), and Takarazuka Kinen\nClear all objectives and complete the training</size>"
 "作戦「追込」で皐月賞でラストの直線で4人以上を抜いて勝利し、\n作戦「追込」で天皇賞（春）を勝利し、\n基礎能力[根性]が1200以上になる。","<size=22>Win the Satsuki Sho as a chaser by breaking through ≥4 people in the last straight\nWin the Tenno Sho (Spring) as a chaser\nReach ≥1200 guts</size>"
 "皐月賞を3 1/2バ身差以上で勝利し、\n日本ダービーを5バ身差以上で勝利し、\n菊花賞を7バ身差以上で勝利し、有馬記念を二連覇する","<size=22>Win the Satsuki Sho, Japan Derby, and Kikka Sho with\ndifference of ≥3 1/2 lengths, ≥5 lengths, and ≥7 lengths\nand win the Arima Kinen twice</size>"
+"皐月賞、菊花賞、有馬記念（クラシック級）、天皇賞（春）を\n作戦「逃げ」で勝利し、うち菊花賞はスタート200m地点から\nゴールまで1番手のままで勝利する。","<size=22>As a runner, win the Satsuki Sho, Kikka Sho, Arima Kinen (2nd year), and Tenno Sho (Spring).\nIn the Kikka Sho, need to stay in the 1st position 200m from start until finish line</size>"
```

Comment:
The "need to" at the end can be removed here to shorten it a bit and still stay accurate.